src/js/scroll_panel.js

```diff
@@ -402,6 +402,10 @@ export class RegularVirtualTableViewModel extends HTMLElement {
             this._invalid_schema = false;
         }
 
+        // polyfill `overflow: overlay` by adding scrollbar dims to table_clip
+        this._table_clip.style.height = `calc(100% + ${this.offsetHeight - this.clientHeight}px)`;
+        this._table_clip.style.width = `calc(100% + ${this.offsetWidth - this.clientWidth}px)`;
```

Comment:
If we're going to generate these in Javascript ... don't use `calc()` also.  Either way, this is specific to the `material` theme and doesn't belong here.

Reply:
What I wanted to calculate here is the dims of `table_clip` if the scrollbars didn't exist:

```
(full width/height of `<regular-table>` elem) - (width/height of `<regular-table>` elem's padding) + (width/height of any scrollbars on the `<regular-table>` elem)
```

Is that part right, at least?

I get that `calc()` is a bit out of place here. The reason I used it is to get the dims of the `<regular-table>` minus any padding. Is there a good js-only way to get that value?